src/js/edition-switch.js

```diff
@@ -4,6 +4,8 @@ const defaultClassName = 'o-header__edition-switch';
 class EditionSwitcher {
 	constructor(headerEl, config = {editionswitchClassName: defaultClassName}) {
 		this.editionSwitcherEl = headerEl.querySelector(`.${config.editionswitchClassName}`);
+		if (!this.editionSwitcherEl) return;
```

Comment:
I only use it in this specific case where there is a `return`. I find it looks cleaner and more readable, but I'm happy to use curly braces everywhere